source/operations.rst

```diff
@@ -0,0 +1,354 @@
+Service Operations
+------------------
+
+In this section you can find the common operational activities related to keep
+the service available to our users.
+
+Initial set-up
+==============
+
+Notebooks VO
+::::::::::::
+
+The resources used for the Notebooks deployments are managed with the 
+``vo.notebooks.egi.eu`` VO. Operators of the service should join the VO, check
+the entry at the `operations portal <https://operations-portal.egi.eu/vo/view/voname/vo.notebooks.egi.eu>`_
+and at `AppDB <https://appdb.egi.eu/store/vo/vo.notebooks.egi.eu>`_.
+
+Clients installation
+::::::::::::::::::::
+
+In order to manage the resources you will need these tools installed
+on your client machine:
+
+* ``egicli`` for discovering sites and managing tokens,
+
+* ``terraform`` to create the VMs at the providers,
+
+* ``ansible`` to configure the VMs and install kubernetes at the providers,
+
+* ``terraform-inventory`` to get the list of hosts to use from terraform.
+
+
+Get the configuration repo
+::::::::::::::::::::::::::
+
+All the configuration of the notebooks is stored at a git repo available in 
+keybase, start by cloning the repo:
```

Comment:
should probably document that membership to a special team is required?

Reply:
Yes, will update.

Anyway, I'm not super-convinced of the keybase approach, would rather have secrets only stored somewhere (which are very few things, 2-3 keys and the check-in clients) and then have the config in GitHub as we could do PR based changes

Reply:
yes Keybase is a temporary solution/workaround before having a cleaner way of doing this